packages/tools/src/tools/annotation/LivewireContourTool.ts

```diff
@@ -798,6 +778,89 @@ class LivewireContourTool extends ContourSegmentationBaseTool {
         },
       }
     );
+    return annotation;
+  }
+
+  /**
+   * Clears any in progress edits, mostly used to get rid of accidentally started
+   * contours that happen on clicking not quite the right handle point.
+   */
+  public deleteInProgress(element, config, evt) {
```

Comment:
Doesn't this sound like an undo? Should we name it undo and mention that we'll have a proper undo feature in the future?